### ISSUE TITLE
TypeError: check_cv() takes from 0 to 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by zard444** [TeamHG-Memex/eli5#414]
Code:
```python
from eli5.sklearn import PermutationImportance
from sklearn.linear_model import LogisticRegression
import numpy as np
X = np.array([1,2,3,4]).reshape(-1,1)
y = [0,1,0,1]
model=LogisticRegression().fit(X,y)
PermutationImportance(model, cv=3).fit(X, y)
```

Error:
```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_15024/2911869046.py in <module>
      5 y = [0,1,0,1]
      6 model=LogisticRegression().fit(X,y)
----> 7 PermutationImportance(model, cv=3).fit(X, y)

~\AppData\Local\Programs\Python\Python310\lib\site-packages\eli5\sklearn\permutation_importance.py in fit(self, X, y, groups, **fit_params)
    200 
    201         if self.cv not in (None, "prefit"):
--> 202             si = self._cv_scores_importances(X, y, groups=groups, **fit_params)
    203         else:
    204             si = self._non_cv_scores_importances(X, y)

~\AppData\Local\Programs\Python\Python310\lib\site-packages\eli5\sklearn\permutation_importance.py in _cv_scores_importances(self, X, y, groups, **fit_params)
    212     def _cv_scores_importances(self, X, y, groups=None, **fit_params):
    213         assert self.cv is not None
--> 214         cv = check_cv(self.cv, y, is_classifier(self.estimator))
    215         feature_importances = []  # type: List
    216         base_scores = []  # type: List[float]

TypeError: check_cv() takes from 0 to 2 positional arguments but 3 were given
```

Here's a very basic example, i've tried googling and documentations for hours to no avail perhaps i don't have enough knowledge about defining self function.

Happens when cv parameter added to PermutationImportance.
Any suggestions on how to fix would be greatly appreciated, thanks!

**Comment by johncant** [comment]
Seems like this repo is more up-to-date: https://github.com/eli5-org/eli5

This issue should be fixed in `master` based on this line: https://github.com/eli5-org/eli5/blob/master/eli5/sklearn/permutation_importance.py#L214

**Comment by zard444** [comment]
Thanks seems your right it is updated, still having issues but will check over there
